examples/coap-server.c

```diff
@@ -517,15 +550,26 @@ main(int argc, char **argv) {
 
   clock_offset = time(NULL);
 
-  while ((opt = getopt(argc, argv, "A:g:p:v:l:")) != -1) {
+  /* Preset default key */
+  key_length = sizeof("secretPSK")-1;
```

Comment:
As said above, I would rather make the default key empty and abort with an error if no key has been provided for PSK.

Reply:
Updated to remove the default key in its entirety.  If no key or cert provided, the server starts in only UPD + TCP mode with no TLS support.